### DISCUSSION TITLE
Spam/advertising deletion?

### DISCUSSION THREAD

**Discussion by Peepo1** [tosdr/tosdr.org#236] [Q&A] [answered]
I know the site wants to be transparent and keep open records of everything but I don't see how these comments contribute to anything. Is there an option to delete them or the accounts associated with them? The spam comments are ugly, unprofessional, and could damage a computer if someone clicks on the hyperlink.

<img width="1280" alt="Screen Shot 2020-05-13 at 5 45 26 PM" src="https://user-images.githubusercontent.com/58347559/81873674-dbb97a00-9541-11ea-8f7c-ce8d100028d6.png">

**Comment by madoleary** [comment]
Hello! This is indeed worrisome. I can assure you that preventing this type
of spam is high on our priorities list. In the short term, we can delete
these comments through manual interaction with the database. In the long
term, we will be implementing a mechanism to mark these sorts of
contributions as spam, to then be deleted. Including myself, the developers
who work on this are volunteers, and all of us have full-time day jobs
outside of this project, so we appreciate your patience, as it may take us
a bit of time to address these issues. Thank you!

**Comment by madoleary** [comment, marked as answer]
Hello, just wanted to follow up. These comments and a large number of others, as well as the users associated with them, have been removed. I've also deployed a mechanism available to curators and admins for flagging spam in the comments. Once flagged, they'll be automatically deleted by a process that runs once a day. I'm going to leave this issue open, and if you notice more spam, you can post links to it here.

**Comment by madoleary** [comment]
I will also be spending some time today looking through the website for additional spam, Thanks, again.

**Comment by Peepo1** [comment]
No problem! Thanks for implementing a solution!
